app/frontend/src/features/communities/leaveReference/UserToReference.tsx

```diff
@@ -0,0 +1,47 @@
+import { Card, Typography } from "@material-ui/core";
+import Avatar from "components/Avatar";
+import BarWithHelp from "components/Bar/BarWithHelp";
+import Divider from "components/Divider";
+import {
+  COMMUNITY_STANDING,
+  COMMUNITY_STANDING_DESCRIPTION,
+  VERIFICATION_SCORE,
+  VERIFICATION_SCORE_DESCRIPTION,
+} from "features/constants";
+import { useStyles } from "features/profile/view/Overview";
+import { LabelsReferencesLastActive } from "features/user/UserTextAndLabel";
+import { User } from "pb/api_pb";
+
+interface UserToReferenceProps {
+  user: User.AsObject;
+}
+
+export default function UserToReference({ user }: UserToReferenceProps) {
```

Comment:
I think, instead of creating a new component for this, turn the one in the profile page into a component, and then use that component here too. It will be more maintainable. 

"Extra" things in the profile page are just the action buttons I think? Those can be children of the new component.